MyLibs/writeOutFish.h

```diff
@@ -25,6 +25,13 @@
 * Created on: Jan. 1, 2018
 */
 
+// TODO: redefine this class, think it clear the relationship between
+// this WriteOut class and the ExperimentData class
+// TODO: unify the input from one place (file, script or command window)
+// TODO: create a yaml config file and corresponding functions
+// TODO: rewrite the WriteOutData class as the FileIO class
```

Comment:
How do you think about this point? Is it necessary to recreate this writeOut class as the FileIO class? 
@WanzhuoShi